src/features/courses/course-controller.ts

```diff
@@ -341,12 +345,24 @@ class CourseController {
                             deadDate: createdCourse.end,
                             partialExtend: false
                         });
+
+                        const topicAssessmentInfo = await curriculumTopic.getCurriculumTopicAssessmentInfo();
+
+                        if (!_.isNil(topicAssessmentInfo)) {
+                            await TopicAssessmentInfo.create({
+                                ..._.omit(topicAssessmentInfo.get({plain: true}), ['id']),
```

Comment:
I feel like this is a place I would prefer verbosity rather than efficiency
It's not as readable and I would rather this broke more royally in the future if additional fields are added rather than a subtle overwrite that causes issues

Reply:
By verbosity, you mean, using a `pick` instead of an `omit`?

Reply:
Yup, I'd rather white list than black list, and I like having that listing in a central place